### ISSUE TITLE
Multiple IfWinActive

### ISSUE THREAD

**Issue by Magikh42** [phil294/AHK_X11#97]
Hello, Is it possible to match multiple class using WinActive

AutoHotkey example

``` autohotkey
#If WinActive("ahk_exe Wow.exe") || WinActive("ahk_exe WowClassic.exe")
```

This if using `#If` which is (i think) not implemented in ahk_x11
I'm using `IfWinActive ahk_class <class>` but this can match only one window

**Comment by phil294** [comment]
no, not possible directly - please know that not only do we not support `#If`, we also don't support `#IfWinActive` (the directive), only `IfWinActive` (the command).

You can, however, write your custom logic.

```ahk
ifwinactive x
{
  ifwinactive y
    {
      ...
    }
}
```
